### PR TITLE
3.6update

### PR DESCRIPTION
**Pull request by a-guiducci**
### What's Changed?
updated the client for the 3.6 release.

My formatter decided to go wild so apologies for the extra line changes for no reason.

- list of changes
Updated accuracy modes to follow 3.6 paradigm
Added requirements to the build's dependencies
Updated unit tests

### PR Checklist

- [x] Updated unit tests
- [x] Ran unit tests

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
 ]
+dependencies = ["requests~=2.31.0"]
```
Comment (a-guiducci) [inline pyproject.toml:20]: Apparently there is a way to do this dynamically and reference the requirements file but I couldn't get it to work and don't have the capacity to dive into it right now. We can add it at a later date

**Comment by yaysummeriscoming** [comment]
If we could get this approved, merged and up on Pypi tomorrow (Friday) that would be great

File: pyproject.toml (hunk 1 above)
Comment (zsolt-pai) [reply pyproject.toml:20]: Adding this to https://privateai.atlassian.net/browse/DEID-2283

**Review by zsolt-pai — APPROVED** [review]
Wonderful!

**Pull request merged**